src/test/java/guitests/UITest.java

```diff
@@ -337,4 +337,11 @@ public GuiTest type(String text) {
         return this;
     }
 
+    /**
+     * Used to select the whole filter text so that it can be replaced
+     */
+    public void selectAll() {
+        pushKeys(KeyCode.END);
```

Comment:
OH. Haha. I thought I am still pressing sift+home. Alright, will remove.
On Nov 10, 2015 1:34 PM, "Darius Foo" <notifications@github.com> wrote:

> In src/test/java/guitests/UITest.java
> <https://github.com/HubTurbo/HubTurbo/pull/1123#discussion_r44371650>:
>
> > @@ -337,4 +337,11 @@ public GuiTest type(String text) {
> >          return this;
> >      }
> >
> > +    /**
> > +     * Used to select the whole filter text so that it can be replaced
> > +     */
> > +    public void selectAll() {
> > +        pushKeys(KeyCode.END);
>
> But whether or not it's at the end, wouldn't Ctrl+A select everything? I
> was just wondering why it was needed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HubTurbo/HubTurbo/pull/1123/files#r44371650>.
>